README.md

```diff
@@ -58,6 +58,15 @@ If you would like to run Sara on your website, follow the instructions
 [here](https://github.com/mrbot-ai/rasa-webchat) to place the chat widget on
 your website.
 
+## To test Sara:
```

Comment:
Great addition to the README!

Reply:
Thanks

Reply:
@rgstephens , I noticed that the url you use `https://github.com/mrbot-ai/rasa-webchat` redirects to this `https://github.com/botfront/rasa-webchat`. Might be better to use the `botfront` version?

Reply:
Thanks, fixed